event_webhook_types.go

```diff
@@ -96,6 +96,8 @@ type CommitCommentEvent struct {
 		CommitID     string `json:"commit_id"`
 		NoteableID   int    `json:"noteable_id"`
 		System       bool   `json:"system"`
+		Description  string `json:"description"`
+		URL          string `json:"url"`
```

Comment:
I'm fine with adding this, but please pace these two below the `StDiff` struct to keep things consistent. Other then that this looks fine 👍🏻

Reply:
Thanks for the feedback, done!